code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -420,6 +420,30 @@
 
 			H.update_icons()
 
+/datum/reagent/blood/reaction_animal(var/mob/living/simple_animal/M, var/method = TOUCH, var/volume)
+
+	var/datum/reagent/blood/self = src
+	if(..())
+		return 1
+
+	if(ismouse(M))//for now, only mice can be infected.
```

Comment:
Would be optimal to have a global list() that has all infectable mob/livings, goes back to my comment above, so that you could just input a new animal path in the singleton and have it become a virology-enabled animal with little effort.

Reply:
there is a proc for that, I had forgotten about it.